### ISSUE TITLE
资源使用问题

### ISSUE THREAD

**Issue by DongDian455** [28598519a/Aeon_AssetDecDL#11]
想问下unity3d文件是怎么预览的？目前通过替换本地文件比较麻烦

**Comment by 28598519a** [comment]
不是很懂你的問題

**Comment by whynotwork** [comment]
What do you do once you click decrypt on spine folder? The files are still unity3d?

**Comment by DongDian455** [comment]
> What do you do once you click decrypt on spine folder? The files are still unity3d?

yes, Is there any way to preview unity3d?

**Comment by DongDian455** [comment]
> 不是很懂你的問題

就是我使用程序下载资源后，资源的文件是unity3d后缀，我没法查看，然后我就把这个文件替换到应用的缓存目录里面去预览，但貌似也预览不了

**Comment by whynotwork** [comment]
> > 不是很懂你的問題
> 
> 就是我使用程序下载资源后，资源的文件是unity3d后缀，我没法查看，然后我就把这个文件替换到应用的缓存目录里面去预览，但貌似也预览不了

Did you figure it out?

**Comment by DongDian455** [comment]
> > > 不是很懂你的問題
> > 
> > 
> > 就是我使用程序下载资源后，资源的文件是unity3d后缀，我没法查看，然后我就把这个文件替换到应用的缓存目录里面去预览，但貌似也预览不了
> 
> Did you figure it out?

No, I have no idea how to preview the file. Is there a tutorial or guide available?

**Comment by bomberman20082009** [comment]
> > > > 不是很懂你的問題
> > > 
> > > 
> > > 就是我使用程序下载资源后，资源的文件是unity3d后缀，我没法查看，然后我就把这个文件替换到应用的缓存目录里面去预览，但貌似也预览不了
> > 
> > 
> > Did you figure it out?
> 
> No, I have no idea how to preview the file. Is there a tutorial or guide available?

解包unity3d文件，里边一般是图像或者spine或者live2d资源，一般是.png .skel  .atlas .json之类的是游戏的立绘或者背景图片或者骨骼。图片直接查看，骨骼一般unity安装相应版本的spine运行库然后查看或录制，或者购买spine软件进行编辑和查看。部分可能是live2d，基本同spine骨骼的查看方式。
